gcloud/datastore/demo/demo.py

```diff
@@ -76,18 +76,18 @@
 
 # You can also work inside a transaction.
 # (Check the official docs for explanations of what's happening here.)
-with datastore.Transaction():
+with datastore.Transaction() as xact:
     print('Creating and saving an entity...')
     key = datastore.Key('Thing', 'foo')
     thing = datastore.Entity(key)
     thing['age'] = 10
-    thing.save()
+    xact.put(thing)
```

Comment:
Maybe I'm mistaken but it seems that `datastore.put([thing])` will work fine here. I feel like the actual surface of `Batch/Transaction` can safely remain opaque to demo users.